save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -46,22 +46,87 @@ class OrganizationService(
      * @return deleted organization
      */
     @Suppress("UnsafeCallOnNullableType")
-    fun deleteOrganization(organizationName: String): Organization = getByName(organizationName)
+    fun deleteOrganization(organizationName: String): Organization = changeOrganizationStatus(organizationName, OrganizationStatus.DELETED)
```

Comment:
what's about validation that all projects are deleted before deleting an organization?